WearTilesKotlin/app/src/debug/java/com/example/wear/tiles/golden/GoldenTilesPreviewsRow3.kt

```diff
@@ -61,6 +62,7 @@ fun News(context: Context) = TilePreviewData {
             it.deviceConfiguration,
             headline = "Millions still without power as new storm moves across US",
             newsVendor = "The New York Times",
+            date = LocalDate.now().minusDays(1),
```

Comment:
I can see the appeal of both options, but is it worth using a fixed date for previews.

Particularly given future screenshot testing of previews?

Reply:
@yschimke 
I think you are exactly right!
I want to display yesterday, I changed it using a fixed Clock.

<img width="633" alt="image" src="https://github.com/user-attachments/assets/405e45f6-2098-4533-9a91-752dc8ce6e53">